src/ape/utils/os.py

```diff
@@ -204,9 +204,18 @@ def run_in_tempdir(
     Args:
         fn (Callable): A function that takes a path. It gets called
           with the resolved path to the temporary directory.
+        name (Optional[str]): Optionally name the temporary directory.
 
     Returns:
         Any: The result of the function call.
     """
     with create_tempdir(name=name) as temp_dir:
         return fn(temp_dir)
+
+
+def clean_path(path: Path) -> str:
+    home = Path.home()
+    if path.is_relative_to(home):
+        return f"{path.relative_to(home)}"
```

Comment:
Does this do `~/`?

Reply:
It does this:

```python
>>> path
PosixPath('/Users/jules/ApeProjects')
>>> f"{path.relative_to(home)}"
'ApeProjects'
```

But I support changing this to include a prefix, like:

```
'~/ApeProjects'
```

or

```
'$HOME/ApeProjects'
```
(more OS agnostic maybe)

Reply:
yeah `$HOME` is good, either is better than no path leader, which could be confusing for some

Reply:
https://github.com/ApeWorX/ape/pull/1931/commits/7b498c1512f9da57273b4a31655e40c9a0349646